### PR TITLE
Fix issues with logout

### LINKED ISSUE #452: Logging out in profile deletes main profile
### Describe the bug
Logging out from a profile removes the login token of the default profile

### To Reproduce
1. Log in in default profile
2. Close Nheko
3. Login in other profile
4. Logout in other profile
5. Close Nheko
6. Open Nheko with main profile
7. Notice, that you need to sign in again.

### Expected behavior
Main profile is still signed in.

### System:

- Nheko version: 0.8.1
- Profile used: Yes
- Installation method: local
- Operating System: Gentoo
- Qt version: 5.15.2

### PR DESCRIPTION
**Pull request by LorenDB**
This fixes the logout issue reported by @deepbluev7, switches to UserSettings where possible, and fixes a bug where the profile actually didn't get set to the default when nheko was run with "-p default".

Fixes #452.

### CODE REVIEW

File: src/ChatPage.cpp [hunk 1]
```diff
@@ -448,15 +448,13 @@ void
 ChatPage::deleteConfigs()
 {
         QSettings settings;
+
+        QString profilePrefix = (UserSettings::instance()->profile() == "default" ? "" : QString("profile/%1").arg(UserSettings::instance()->profile()));
+        settings.beginGroup(profilePrefix);
```
Comment (deepbluev7) [inline src/ChatPage.cpp:453]: Shouldn't this be in an if instead? or does beginGroup("") work?

Comment (LorenDB) [reply src/ChatPage.cpp:453]: Good point, I'm not sure

Comment (deepbluev7) [reply src/ChatPage.cpp:453]: Let's just go with the safe variant, aka an if then :3

**Comment by deepbluev7** [comment]
Yep, works now!

**Comment by deepbluev7** [comment]
Thank you for the fast fix!

**Pull request merged**